utils/enphase.js

```diff
@@ -0,0 +1,27 @@
+const fetch = require('node-fetch'); // Import fetch (for network requests)
+const moment = require('moment');
+
+// SEE ENPHASE API DOCS HERE https://developer.enphase.com/docs
+// Returns object with `production`, `meter_production`, `micro_production` keys with production data
+const getEnphaseData = async (userId, systemId, startDate, endDate) => {
+  // Build API Url
+  const baseUrl = 'https://api.enphaseenergy.com/api/v2';
+  const key = process.env.ENPHASE_KEY;
```

Comment:
nit: should we move this to constants to the top of the file or to a new constants file? 

i personally think that moving it to a new constants file might be too heavy of an approach at this point, but maybe moving it to the top?

Reply:
Sure sure